.circleci/config.yml

```diff
@@ -0,0 +1,71 @@
+version: 2.1
+
+executors:
+  node:
+    docker:
+      - image: node:12
+        user: node
+  gcloud:
+    docker:
+      - image: google/cloud-sdk:alpine
+
+commands:
+  deploy-env:
+    description: "Deploy to a specific environment"
+    parameters:
+      sa_key_var:
+        type: string
+        default: "SA_KEY_JSON"
+      env:
+        type: string
+        default: "dev"
+      pr:
+        type: boolean
+        default: false
+    steps:
+      - attach_workspace:
+          at: .
+      - run: cp config/<< parameters.env >>.json config.json
+      - run: echo "$<< parameters.sa_key_var >>" > /tmp/sa.json
+      - run: gcloud auth activate-service-account --key-file=/tmp/sa.json
+      - run: gcloud app deploy --project=terra-bard-<< parameters.env >> --promote --quiet
+
+jobs:
+  build:
+    docker:
+      - image: node:12
+    steps:
+      - checkout
+      - restore_cache:
+          key: v1-{{ checksum "yarn.lock" }}
+      - run: yarn install --frozen-lockfile
+      - save_cache:
+          key: v1-{{ checksum "yarn.lock" }}
+          paths:
+            - node_modules
+            - ~/.cache/yarn
```

Comment:
Caching both has turned out to be more trouble than it's worth, IMO, in terms of how long it takes to restore all of those folders and sub-folders. I recommend just doing the `.cache`.